pkg/apis/nsx.vmware.com/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -9,8 +9,13 @@ import (
 )
 
 const (
-	AccessModePublic  string = "Public"
-	AccessModePrivate string = "Private"
+	AccessModePublic    string = "Public"
+	AccessModePrivate   string = "Private"
+	AccessModeProject   string = "Project"
+	LbServiceSizeSmall  string = "SMALL"
```

Comment:
NSX no longer calls a LB Instance LB Service in a VPC. It is just called LoadBalancer.

So, please remove `Service `when referring to a LB Instance.